### ISSUE TITLE
Videos open much slower than expected. 

### ISSUE THREAD

**Issue by Schweiger-Jacob** [mpv-player/mpv#11391]
### Important Information

Provide following Information:
- mpv version 0.34.1
- Linux Distribution and Version linix mint 21.1
- Source of the mpv binary software center 
- If known which version of mpv introduced the problem
- Window Manager and version mutter 
- GPU driver and version intel power saving mode 
- Possible screenshot or video of visual glitches 

ffmpeg] AVHWDeviceContext: Failed to query surface attributes: 20 (the requested function is not implemented).
[vo/gpu/vaapi-egl] failed to retrieve libavutil frame constraints

If you're not using git master or the latest release, update.
Releases are listed here: https://github.com/mpv-player/mpv/releases

### Reproduction steps

Try to reproduce your issue with --no-config first. If it isn't reproducible
with --no-config try to first find out which option or script causes your issue.

Describe the reproduction steps as precise as possible. It's very likely that
the bug you experience wasn't reproduced by the developer because the workflow
differs from your own.

### Expected behavior
video opens in less than a second or so 
### Actual behavior
video takes more than 6 seconds to open (i timed it =)
### Log file

Make a log file made with -v or --log-file=output.txt, paste it to
https://0x0.st/ or attach it to the github issue, and replace this text with a
link to it.

The issue will be closed for ignoring the issue template.

### Sample files

Sample files needed to reproduce this issue can be uploaded to https://0x0.st/
or similar sites. (Only needed if the issue cannot be reproduced without it.)
Do not use garbage like "cloud storage", especially not Google Drive.

**Comment by CounterPillow** [comment]
Please use `--log-file`, not `-v`, so that it includes timestamps for the log lines.

**Comment by Schweiger-Jacob** [comment]
[log.txt](https://github.com/mpv-player/mpv/files/10858997/log.txt)

**Comment by CounterPillow** [comment]
the problem is here:
```
[   0.265][v][vo/gpu] Loading hwdec driver 'cuda-nvdec'
[   5.027][v][vo/gpu/cuda-nvdec] cu->cuInit(0) failed -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
```
mpv is probing which hwdec interop to use for gpu<->hwdec interop. This leads to it loading some CUDA related driver still on your system that is very slow to load (as is expected of anything nvidia).

To confirm this, try running with the option `--gpu-hwdec-interop=vaapi`.

**Comment by Schweiger-Jacob** [comment]
> 

I got 

No hwdec backend named 'vaapi' found!
Error parsing option gpu-hwdec-interop (option parameter could not be parsed)
Setting commandline option --gpu-hwdec-interop=vaapi failed.

as the response

**Comment by Schweiger-Jacob** [comment]
> the problem is here:
> 
> ```
> [   0.265][v][vo/gpu] Loading hwdec driver 'cuda-nvdec'
> [   5.027][v][vo/gpu/cuda-nvdec] cu->cuInit(0) failed -> CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
> ```
> 
> mpv is probing which hwdec interop to use for gpu<->hwdec interop. This leads to it loading some CUDA related driver still on your system that is very slow to load (as is expected of anything nvidia).
> 
> To confirm this, try running with the option `--gpu-hwdec-interop=vaapi`.
I got

No hwdec backend named 'vaapi' found!
Error parsing option gpu-hwdec-interop (option parameter could not be parsed)
Setting commandline option --gpu-hwdec-interop=vaapi failed.

as the response

**Comment by Dudemanguy** [comment]
It's `--gpu-hwdec-interop=vaapi-egl` in your version of mpv. The name changed later.

**Comment by Schweiger-Jacob** [comment]
> It's `--gpu-hwdec-interop=vaapi-egl` in your version of mpv. The name changed later.

Thank you that worked. Videos now open very fast.

**Comment by Dudemanguy** [comment]
FYI, the slow probing is fixed in 0.35 so you won't need that option on the next version.
